diff --git a/src/routes/+page.svelte b/src/routes/+page.svelte
index c71bcd1..fcfec57 100644
--- a/src/routes/+page.svelte
+++ b/src/routes/+page.svelte
@@ -1,6 +1 @@
-<script>
-  import { base } from '$app/paths';
-</script>
-
 <h1>SvelteKit Web3 Login</h1>
-<a href="{base}/about">About</a>
diff --git a/src/routes/about/+page.svelte b/src/routes/about/+page.svelte
index ed1df81..ae068f6 100644
--- a/src/routes/about/+page.svelte
+++ b/src/routes/about/+page.svelte
@@ -1,7 +1 @@
-<script>
-  import { base } from '$app/paths';
-</script>
-
 <h1>About</h1>
-<p>A page to test Web3 login on SvelteKit.</p>
-<a href="{base}/">Home</a>
